checkQC/app.py

```diff
@@ -45,10 +46,11 @@ class App(object):
     This is the main application object for CheckQC.
     """
 
-    def __init__(self, runfolder, config_file=None, json_mode=False):
+    def __init__(self, runfolder, config_file=None, json_mode=False, downgrade_error=()):
```

Comment:
I think it would be better to call this `downgrade_errors_for` or possibly `downgrade_errors_list`, so that it is easier to understand that this is something that happens for specific handlers, and not for all of them.

Reply:
I also think that it is clearer to set the default to `None`.

Reply:
I updated the variable name, good suggestion! 
I did not update the default value, an iterable is needed. I thought it would be cleaner to do this than to convert the None value to a empty tuple in the code. Maybe you can comment again on the new commits if you disagree.

Reply:
No, this looks good. I wanted to avoid the problem described here: https://effbot.org/zone/default-values.htm However, I had forgotten that tuple in python is immutable, so in this case it should be ok, unless I misunderstand something. So let's stick with this. :+1: